docs/cdm-guidelines.md

```diff
@@ -43,6 +43,9 @@ needs to be addeed to the project pom.xml:
 -   The CDM uses [builder pattern](https://en.wikipedia.org/wiki/Builder_pattern) for each of the pojos. The
     distribution ships with the json to java object serialisers.
 
+_NOTE:_ For CDM releases prior to version 4.0.0 the dependencies can be found in the ISDA repository: `https://europe-west1-maven.pkg.dev/production-208613/isda-maven`
```

Comment:
Just to be fully explicit, should we state that current releases are available in Maven Central.  Something like:
> All current CDM dependencies are available in Maven Central.  For CDM releases prior to version 4.0.0 the dependencies can be found in the ISDA repository: